clean_cache/action.yml

```diff
@@ -0,0 +1,90 @@
+name: 'clean_cache'
+description: 'clean the cache'
+inputs:
+  s3cmd:
```

Comment:
Where in this action is this parameter used. And what do you want to accomplish with it?

Reply:
its wrong